pkg/cke/main.go

```diff
@@ -27,6 +27,8 @@ var (
 	flgSessionTTL           = pflag.String("session-ttl", "60s", "leader session's TTL")
 	flgDebugSabakan         = pflag.Bool("debug-sabakan", false, "debug sabakan integration")
 	flgMaxConcurrentUpdates = pflag.Int("max-concurrent-updates", 10, "the maximum number of components that can be updated simultaneously")
+	flgDrainRetryTimes      = pflag.Int("drain-retry-times", 30, "number of drain retries")
```

Comment:
This includes the initial attempt.
This way of counting differs from `reboot.command_retries` in cluster.yaml.
This may be confusing.

To distribute configuration parameters to cluster.yaml and the command flags may also be confusing.

Reply:
It would be nice to describe the retry behavior in docs/reboot.md and/or docs/cluster.md.

Reply:
I got confused by seeing `eviction_timeout_seconds` configuration parameter.